.github/workflows/pr-workflow.yml

```
@@ -188,15 +239,30 @@
+          make docker.build
+
+          # Save image to cache
+          docker save $ECR_NAME:$COMMIT | gzip > /tmp/fetcher-image.tar.gz
+          echo "image=$ECR_NAME:$COMMIT" >> $GITHUB_OUTPUT
+
+      - name: Cache Docker image
+        uses: actions/cache@v4
+        with:
+          path: /tmp/fetcher-image.tar.gz
+          key: fetcher-image-${{ github.sha }}
+          retention-days: 7
```

actionlint (1.7.7)

input "retention-days" is not defined in action "actions/cache@v4". available inputs are "enableCrossOsArchive", "fail-on-cache-miss", "key", "lookup-only", "path", "restore-keys", "save-always", "upload-chunk-size"

(action)